acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -1,6 +1,14 @@
 def initialize_repo_on_host(host, os, nightly)
   case os
   when :debian
+
+    # For openjdk8
+    if host.platform.start_with?('debian-8')
```

Comment:
Ahh -- there's actually an (improperly indented?) update at the end of this case category.